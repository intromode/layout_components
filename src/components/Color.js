import React from 'react';

export default function Color() {
  return (
    <dl>
      <dt>Name</dt>
      <dd>Purple</dd>
  
      <dt>hex</dt>
      <dd>#123456</dd>
  
      <dt>rgb</dt>
      <dd>rgb(221,160,221)</dd>
    </dl>

  );
}
