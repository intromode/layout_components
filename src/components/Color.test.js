import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color.js';

describe('testing header component', () => {
  it('matches a screenshot', () => {
    const wrapper = shallow(<Color />);
    expect(wrapper).toMatchSnapshot();
  });
});
