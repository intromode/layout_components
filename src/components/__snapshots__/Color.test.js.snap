// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing header component matches a screenshot 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Purple
  </dd>
  <dt>
    hex
  </dt>
  <dd>
    #123456
  </dd>
  <dt>
    rgb
  </dt>
  <dd>
    rgb(221,160,221)
  </dd>
</dl>
`;
